cit-api/pipeline/importers/csv_resource.py

```diff
@@ -59,7 +61,7 @@ def import_resource(resource_type):
         for row in data:
             model_class = apps.get_model("pipeline", data_source.model_name)
             import_data_into_point_model(resource_type, model_class, row)
-        calculate_nearest_location_type_outside_50k(resource_type)
+        # calculate_nearest_location_type_outside_50k(resource_type)
```

Comment:
IIRC we want this off in production right?

Reply:
It is a part of the import process that just takes to longe to realistically run. So I found it in here when I was adding in this new CSV because most of our location imports are now done using WMS.